experimental/generation/generator/packages/library/templates/standard/luis.lu.dialog.lg

```diff
@@ -14,10 +14,13 @@
     "endpointKey": "=settings.luis.endpointKey",
     "predictionOptions": {
         "includeInstanceData": true
-    }
+    },
+    "dynamicLists": "=${addList()}"
 }
 ```
 
 # appName
 - ${prefix}_${replace(locale, '-', '_')}_lu
 
+# addList
+- [${join(foreach(where(schema.properties, property, property.value.$dynamiclist == "true"), result, `dialog.${result.key}Value`), `,`)}]
```

Comment:
>result.key [](start = 111, length = 10)

Is this correct?  I think you need an indicesAndValues around schema.properties, right?